.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
There was no reason. I was wrong. I thought about Symfony3 and its requirements.